dvc/remote/base.py

```diff
@@ -226,6 +439,7 @@ def ospath(self):
         return posixpath
 
     def checksum_to_path(self, checksum):
+        assert checksum
```

Comment:
Maybe we can use [pytype](https://github.com/google/pytype) and remove these assertions.